contrib/config-example

```diff
@@ -0,0 +1,45 @@
+###
+##        Core Lightning Configurations
+###
+##  Edit this file to your desired configurations
+##        Edit this files name to `config`
+###
+##    To view all documentation & options run:
+##          `man lightningd config`
+###
+
+# Node Name / Color
+alias=<NODE-ALIAS> # Call your node whatever you like!
+rgb=FFA500  # Pick your favorite color as a hex code!
+
+# Run lightningd as a daemon (Recommended)
+daemon
+
+# Set the network for Core Lightning to sync to, Bitcoin Mainnet for most users
+# Omit this field if the config file is in a network directory such as `.lightning/bitcoin`
+network=bitcoin
+
+# Channel Options (Set to defaults)
+# !! Please read the manual before editing these and for a full list of available options !!
+fee-base=1000 # Base fee to charge for every payment which passes through in MILLISATOSHI (1/1000 of a satoshi)
+fee-per-satoshi=10 # In millionths (10 is 0.001%, 100 is 0.01%, 1000 is 0.1% etc.)
+min-capacity-sat=10000 # Minimum value, in SATOSHI, to accept for channel open requests
+htlc-minimum-msat=0 # Sets the minimum HTLC value for new channels
+funding-confirms=3 # Blockchain confirmations required for channel to be considered valid
+
+# Bitcoin Settings
+bitcoin-cli=/path/to/your/bitcoin-cli # If unsure run: `whereis bitcoin-cli`
+bitcoin-datadir=/path/to/your/.bitcoin # Bitcoin directory, defaults to `~/.bitcoin`
+bitcoin-rpcconnect=127.0.0.1 # Bitcoin RPC server running on localhost, change to your bitcoind IPADDRESS
+bitcoin-rpcport=8332 # Bitcoin RPC server default port 8332, change to your bitcoind PORT
+bitcoin-rpcuser=<BITCOIND-RPCAUTH-USER>
+bitcoin-rpcpassword=<BITCOIND-RPCAUTH-PASSWORD>
```

Comment:
Mh this is right, but it is just for the core lightning plugin.

Can we add just the essential field here? otherize if you use a different plugin backend you will endup in a strange error, and if you are a new user (if you use a conf example you are) this can be confusing